components/server/src/server.rs

```diff
@@ -863,6 +867,31 @@ impl<ER: RaftEngine> TiKVServer<ER> {
             self.to_stop.push(ttl_checker);
         }
 
+        if self.config.backup.enable_streaming {
+            // Create backup stream.
+            let mut backup_stream_worker = Box::new(LazyWorker::new("br-stream"));
+            let backup_stream_scheduler = backup_stream_worker.scheduler();
+
+            // Register br-stream observer.
+            let backup_stream_ob = BackupStreamObserver::new(backup_stream_scheduler.clone());
+            backup_stream_ob.register_to(self.coprocessor_host.as_mut().unwrap());
+            // Register config manager.
+            cfg_controller.register(
+                tikv::config::Module::Backup,
+                Box::new(BackupStreamConfigManager(backup_stream_worker.scheduler())),
+            );
+
+            let meta_store = EtcdStore::connect(&self.config.pd.endpoints);
```

Comment:
Maybe adapt TLS(or not for now?)

Reply:
Ok, I'll leave a TODO. not introduce in this PR.